include/okapi/api/chassis/model/threeEncoderSkidSteerModel.hpp

```diff
@@ -37,6 +37,13 @@ class ThreeEncoderSkidSteerModel : public SkidSteerModel {
    */
   std::valarray<std::int32_t> getSensorVals() const override;
 
+
+  /**
+   * Reset the sensors to their zero point.
+   */
+  void resetSensors() const override;
+  
+
```

Comment:
Please either fix this whitespace by hand or install `clang-format `and use that.